easybuild/tools/options.py

```diff
@@ -957,7 +969,7 @@ def show_config(self):
 
         # options that should never/always be printed
         ignore_opts = ['show_config', 'show_full_config']
-        include_opts = ['buildpath', 'installpath', 'repositorypath', 'robot_paths', 'sourcepath']
+        include_opts = ['buildpath', 'installpath', 'repositorypath', 'robot_paths', 'search_paths', 'sourcepath']
```

Comment:
I wouldn't include it here to make it always printed, just let it be printed if it's different from the default (works automagically)?